src/pps/pps.proto

```diff
@@ -80,9 +80,7 @@ message Service {
 }
 
 message Spout {
-  bool overwrite = 1;
   Service service = 2;
-  string marker = 3;
```

Comment:
Can you make these reserved, just in case we at some point reuse the field numbers and hit a compatibility issue?

Reply:
Hmm, that's probably fine, in that case might as well change the field number to 1?